reproducible/BUILD

```diff
@@ -32,19 +32,39 @@ pkg_tar(
 load("//reproducible:debootstrap.bzl", "debootstrap_image")
 
 debootstrap_image(
-    name = "debootstrap_base",
+    name = "jessie_base",
+    distro = "jessie",
+)
+
+debootstrap_image(
+    name = "buster_base",
+    distro = "buster",
 )
 
 docker_build(
     name = "debian8",
-    base = ":debootstrap_base",
+    base = ":jessie_base",
+    tars = [
+        ":overlay.tar",
+    ],
+)
+
+docker_build(
+    name = "debian9",
+    base = ":buster_base",
```

Comment:
debian9 is stretch, debian10 is buster. we should have both, but stretch is GA and is obviously the higher priority.

Reply:
D'oh. Added both.